Assets/Scripts/Weapons/Weapon.cs

```diff
@@ -9,6 +9,7 @@ public abstract class Weapon : MonoBehaviour
 
     // The number of seconds between attacks
     public float timeBetweenAttacks;
+    public AudioSource fireAudio;
```

Comment:
I would add this to ProjectileWeapon.cs instead of Weapon.cs. Or maybe just call it `attackAudio`. At this point, though, it doesn't really matter.